client/gutenberg/extensions/visited/index.js

```diff
@@ -0,0 +1,35 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { __, _x } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import edit from './components/edit';
+import save from './components/save';
+import { CRITERIA_AFTER, DEFAULT_THRESHOLD } from './constants';
+import './editor.scss';
+
+export const name = 'visited';
+export const settings = {
+	attributes: {
+		criteria: {
+			type: 'string',
+			default: CRITERIA_AFTER,
+		},
+		threshold: {
+			type: 'number',
+			default: DEFAULT_THRESHOLD,
+		},
+	},
+	category: 'jetpack',
+	description: __( 'Control block visibility based on how often a visitor has viewed the page.' ),
+	icon: 'universal-access-alt',
```

Comment:
I'm not sure why you chose this icon. What do you think about using this one: https://material.io/tools/icons/?icon=history&style=baseline

Reply:
Or the Visibility one: https://material.io/tools/icons/?search=visi&icon=visibility&style=outline

(note: style should be set to "Outlined")